### PR TITLE
#trivial Added swiftlint

### PR DESCRIPTION
**Pull request by MortyMerr**
This PR closes https://github.com/RxSwiftCommunity/RxDataSources/issues/316

### CODE REVIEW

File: Examples/Example/Example2_RandomizedSectionsAnimation.swift [hunk 1]
```diff
@@ -13,19 +13,27 @@ import RxCocoa
 import CoreLocation
 
 class NumberCell : UICollectionViewCell {
-    @IBOutlet var value: UILabel?
+    @IBOutlet private var value: UILabel?
+
+    func configure(value: String) {
```
Comment (freak4pc) [inline Examples/Example/Example2_RandomizedSectionsAnimation.swift]: I'd make these similar to this, to follow Apple naming conventions: 
```suggestion
    func configure(with value: String) {
```

File: Examples/Example/Example5_UIPickerView.swift [hunk 2]
```diff
@@ -13,41 +13,49 @@ import RxDataSources
...
+        components: [],
+        numberOfComponents: { _,_,_  in 1 },
+        numberOfRowsInComponent: { _, _, items, _ -> Int in
+            return items.count
+        },
+        attributedTitleForRow: { _, _, items, row, _ -> NSAttributedString? in
+            NSAttributedString(
+                string: items[row],
+                attributes: [
+                    NSAttributedStringKey.foregroundColor: UIColor.purple,
```
Comment (freak4pc) [inline Examples/Example/Example5_UIPickerView.swift]: `NSAttributedAtringKey` can be removed I believe

File: Examples/Example/Example2_RandomizedSectionsAnimation.swift (hunk 1 above)
Comment (MortyMerr) [reply Examples/Example/Example2_RandomizedSectionsAnimation.swift]: Sure, thanks, fixed

File: Examples/Example/Example5_UIPickerView.swift (hunk 2 above)
Comment (MortyMerr) [reply Examples/Example/Example5_UIPickerView.swift]: Sure, thanks, fixed

**Comment by MortyMerr** [comment]
Some strange things happened with Randomizer. Revert that changes

**Comment by MortyMerr** [comment]
Hey @freak4pc is that ok now? :)

**Comment by freak4pc** [comment]
@MortyMerr please never merge your own PRs without a review. It didn’t have to be me but you can’t just push your own code without a second pair of eyes. Thank you for understanding.

**Comment by MortyMerr** [comment]
@freak4pc as I remember RxSwiftExt invite letter - I can do so. PR can be reverted in any time. 
I requested review from four approvers on that trivial PR, which doesn’t change public API or add new functionality. I didn’t get the review for three months.
I think it’s ok to merge it by myself, cause I fixed all comments from you. 
Thank you for understanding. Sorry if that doesn’t fits RxSwiftExt guidelines. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by freak4pc** [comment]
1. No one of the contributors of this repositories are not hired or paid to review yours or anyone else's contributions. Sometime it takes time, and sometimes you need to ping someone to review.

2. Collaboration doesn't mean "I do whatever I want because I got an invitation to be a member" - Collaboration is waht it means, collaborating with **other** contributors.

You reverted the change and only pinged me once (in July) when I wasn't with a computer, so I wasn't able to review it again. But in general - not even for open source, It's not a good practice to merge your own PR until there's a green approval from at least one contributor. That contributor doesn't have to be me, but it needs to be someone else who works on this project. 

Please respect the fact this is a community project and it's maintained by _many_ people such as yourself (on their spare time), so sometimes reviews take longer. It doesn't mean you can make a single-handed decision on merging content. 

Thank you.

**Pull request merged**